junifer/data/parcellations.py

```diff
@@ -700,3 +700,84 @@ def _retrieve_suit(
     ].to_list()
 
     return parcellation_fname, labels
+
+
+def merge_parcellations(
+    parcellations_list: List,
+    parcellations_names: List,
+    labels_lists: List[List],
+) -> Tuple:
+    """Merge all parcellations from a list into one parcellation.
+
+    Parameters
+    ----------
+    parcellations_list : list
```

Comment:
The basic types like str can be documented here as well for ease.

Reply:
you mean `parcellations_list : list of Nifti1Image`?

Reply:
Yeah but I think it should be `... niimg-like object` as `nibabel` calls it.

Reply:
ok will try that, i think i put `niimg` before which the docs didn't recognise, but `niimg-like object`  makes sense